tests/pages/customer-resource-coverage.js

```diff
@@ -56,8 +72,8 @@ function createRowObject(element) {
     },
 
     clearBeginDate() {
-      let $clearButton = $('[data-test-eholdings-package-details-custom-begin-coverage]')
-        .find('button[id^=datepicker-clear-button]');
+      let $clearButton = $('[data-test-eholdings-coverage-form-date-range-begin]')
+        .find('button[id^=datepicker-clear-button-undefined]');
```

Comment:
Is `datepicker-clear-button-undefined` really the id? 😨

Reply:
i think this is why we do a fuzzy search instead of a direct search for id - i added the `undefined` back while debugging tests and forgot to go back and yank it out.  furthermore, it's possible this changed after the rebase.  i'll do some digging